runsc/boot/loader.go

```diff
@@ -191,6 +191,9 @@ func New(args Args) (*Loader, error) {
 		return nil, fmt.Errorf("setting up memory usage: %v", err)
 	}
 
+	// Sets the refs leak check mode
+	refs.SetLeakMode(args.Conf.ReferenceLeakMode)
```

Comment:
Not sure if this is the right place to set refs leak mode from config. @fvoznika it would be great if you can provide your feedback

Reply:
Please move even further up, before usage is initialized. Pretty much as early as possible in the boot sequence.